client/src/components/Menu/Menu.scss

```diff
@@ -249,19 +247,18 @@
 }
 
 .cms-menu-mobile-overlay {
+  display: block;
   position: absolute;
-  background-color: rgba(0, 0, 0, 0.05);
+  transition: opacity .3s;
+  transform: translateX(-100%);
+  background-color: rgba(0, 0, 0, 0.25);
   z-index: $zindex-cms-content-header + 1;
   border: 0;
 }
 
-.cms-menu + .cms-menu-mobile-overlay {
-  display: none;
-}
-
 .cms-menu.cms-menu--open + .cms-menu-mobile-overlay {
   @include media-breakpoint-down(sm) {
-    display: block;
+    opacity: 1;transform: translateX(0);
```

Comment:
This should be split onto another line, you can run yarn build and this will also run the linter indicating these errors/warnings. As @bergice mentioned you'll need to update your PR to include the rebuilt bundle files.

![image](https://user-images.githubusercontent.com/24258161/80782272-86887a80-8bc9-11ea-855f-1e2fa949a823.png)

Reply:
Sure, I was wrong the past year, now fixed with the commit [88c4ba7](https://github.com/alessandromarotta/silverstripe-admin/commit/88c4ba70f6bf2dcaf77ee18c70e3f2ca1fb0e30f)

Reply:
![image](https://user-images.githubusercontent.com/24258161/80937985-66f08c80-8e2b-11ea-9f91-2aa0bae4ce01.png)
Looks like you still have one error which is why the builds are failing. 
Once you fix this and run yarn build you'll see the `bundle.css` file which you'll need to push up too. You can see more about how to build the bundle files here: https://docs.silverstripe.org/en/4/contributing/build_tooling/#build-commands